test/integration_test.rb

```diff
@@ -7,13 +7,38 @@ def setup
   end
 
   # Save files in case you'd like to see what the result is
-  def teardown
```

Comment:
As of this new version, styles are indexed on `serialize`. I don't want to use this hook, because we need test some assertions after the spreadsheet is processed.